src/preview.js

```diff
@@ -27,7 +27,23 @@ const getPreviewBody = ( type, message, cta ) => {
 }
 
 const getReadOnWikiCta = ( lang, title, isTouch ) => {
-	return `<a href="${ buildWikipediaUrl( lang, title, isTouch ) }" target="_blank" class="wikipediapreview-cta-readonwiki">${ msg( lang, 'read-on-wiki' ) }</a>`
+	return `<a href="${ buildWikipediaUrl( lang, title, isTouch ) }" target="_blank" class="wikipediapreview-footer-link-cta">${ msg( lang, 'read-on-wiki' ) }</a>`
+}
+
+const getLinkIconSvg = ( dir = 'ltr' ) => {
```

Comment:
For security reasons the rules you can set to `::visited` are very limited, only color related changes basically, so we can't add the svg as `content` for `::after` if we want to achieve the color change

Reply:
Great thanks, `getLinkIconSvg` added to utils and dark mode question brought up to design already